README.md

```diff
@@ -40,3 +40,10 @@ To contribute to this tanslation project, please book your topic by creating an
 | [potpourri.md](https://github.com/missing-semester-cn/missing-semester-cn.github.io/blob/master/_2020/potpourri.md) |  [@catcarbon](https://github.com/catcarbon) | Done |
 | [qa.md](https://github.com/missing-semester-cn/missing-semester-cn.github.io/blob/master/_2020/qa.md) | [@AA1HSHH](https://github.com/AA1HSHH) | Done |
 | [about.md](https://github.com/missing-semester-cn/missing-semester-cn.github.io/blob/master/about.md)  | [@Binlogo](https://github.com/Binlogo)  | Done |
+
+
+
+
+
+hello ! world!
+hello ! git!
```

Comment:
Why do you modify this file? If you just want to test, please revert your changes before merge or your pull request will not be merged.

Reply:
So sorry,I modified that file at first time trying git and forgot to correct it.I’ve got it repaired. @irvingskr

Reply:
I deleted the superfluous content about the `README.md` (local workspace)(the problem you mentioned) , then I `add/commit/pust` it to my  repository(showed my commit here , pull request page).But looks like here is a `"Changes requested"`,which means I also need to modify my homework(?)
But sorry I still couldn't found the problem.(though I 've check other's PR page and search the git Doc on the Internet)
 Could you please tell me about that? 
@irvingskr

Reply:
> 

Looks like you have understood the change request and resolve it，good job！